cluster/terraform_kubernetes/kube_state_metrics.tf

```diff
@@ -0,0 +1,160 @@
+data "kubernetes_namespace" "kube_system" {
+  metadata {
+    name = "kube-system"
+  }
+}
+
+resource "kubernetes_namespace" "kube_system" {
+  count = data.kubernetes_namespace.kube_system.id == null ? 1 : 0
+
+  metadata {
+    name = "kube-system"
+  }
+}
```

Comment:
can we put all resources in the monitoring namespace

Reply:
done

Reply:
Can we drop the data and resource block for the monitoring namespace.
Just set the new resources to use 'namespace = kubernetes_namespace.default_list["monitoring"].metadata[0].name' instead

Reply:
just hardcoded it to monitoring

Reply:
It needs to be 'namespace = kubernetes_namespace.default_list["monitoring"].metadata[0].name' to stop initial cluster build dependency issues